test.js

```diff
@@ -9,3 +10,24 @@ test('valid username', async t => {
 	t.is(await npmEmail('sindresorhus'), 'sindresorhus@gmail.com');
 	t.is(await npmEmail('vdemedes'), 'vadimdemedes@hey.com');
 });
+
+test('non-existent user', async t => {
+	t.is(await npmEmail('nnnope'), undefined);
+});
+
+test('handles 404', async t => {
```

Comment:
I meant the other tests. This one does indeed need to be serial because of the mocking.